### DISCUSSION TITLE
Blade::render(..) with deleteCachedView: true does not delete the compiled view [hash].php. Only the [hash].blade.php view.

### DISCUSSION THREAD

**Discussion by rogiermulders** [laravel/framework#53125] [Ideas]
### Laravel Version

v11.27.2 and v10.48.22

### PHP Version

8.3.6

### Database Driver & Version

MySql (...)

### Description

When I use the Blade::render function with the attribute deleteCachedView set to true,
It only throws away the [hash].blade.php file
NOT the [hash].php file.

This is not what I would expect.
I would expect both files were deleted.

Im using Blade::render to test if a view is valid. Do not want to fill view cache with useless test results.

### Steps To Reproduce

Fresh laravel install.

In /routes/web.php 
```
<?php

use Illuminate\Support\Facades\Blade;
use Illuminate\Support\Facades\Route;

Route::get('/', function () {
    Blade::render('<div>Test A</div>',[]);
    Blade::render('<div>Test B</div>',[], true);   // deleteCachedView: true
    return 'Test run!';
});
```
```
php artisan view:clear
php artisan serve
```
goto http://localhost:8000/

Now I would expect I have no Test B views cached but there still is.

![image](https://github.com/user-attachments/assets/603521be-220e-4bbf-9bf7-5d516d615158)

**Comment by jurgen-vk** [comment]
I noticed this too
